addons/website_slides/models/slide_question.py

```diff
@@ -23,10 +25,10 @@ class SlideQuestion(models.Model):
     @api.constrains('answer_ids')
     def _check_answers_integrity(self):
         for question in self:
-            if len(question.answer_ids.filtered(lambda answer: answer.is_correct)) != 1:
-                raise ValidationError(_('Question "%s" must have 1 correct answer', question.question))
-            if len(question.answer_ids) < 2:
-                raise ValidationError(_('Question "%s" must have 1 correct answer and at least 1 incorrect answer', question.question))
+            if question.correct_answers == 0 or question.wrong_answers == 0:
+                raise ValidationError(_('Question "%s" from "%s" must have at least 1 correct answer and 1 incorrect answer',
+                                        question.question,
+                                        question.slide_id.name))
```

Comment:
You'll find several examples in `eLearning`(I think), or `event`  (I'm sure), where we build the list of validation errors in the loop but then return a message containing all errors at once, to help the user correct them all faster.